config/addons/lfp-batteries/RelayBoardV3Node/remote_config/BeagleBoard.json

```diff
@@ -0,0 +1,39 @@
+{
+	"charge_current_ain": -1,
+	"charge_profile": {
+		"current_offset": 1.4,
```

Comment:
This will change for LFP batteries?

Reply:
No, that value is always needed, so that we know what to send to the charger. There is simply no safe and reliable way to get that value from anywhere. 1.4A is an estimate that's usually right.